pkg/utils/utils.go

```diff
@@ -183,6 +183,11 @@ func DeDuplicate(input []string) []string {
 	result := []string{}
 	internal := map[string]struct{}{}
 	for _, value := range input {
+		value = strings.TrimSpace(value)
```

Comment:
i think empty string is also a `meaningful` value, WDYT?

Reply:
I am afraid I agree with HusterWan.

Reply:
i mean when the func used for deal with labels: like "lable1", " ", "lable2", and some like join with ',' operation, it will get some like 'lable1,,lable2' or 'lable1,lable2,,'